app/Model/Project.php

```diff
@@ -1820,4 +1820,42 @@ public function ComputeTestingDayBounds($date)
         $endOfDay = gmdate(FMT_DATETIME, $end_timestamp);
         return [$beginningOfDay, $endOfDay];
     }
+
+
+    /**
+     * Return the testing day (a string in DATETIME format)
+     * for a given date (a date/time string).
+     */
+    public function GetTestingDay($date)
+    {
+        // If the build was started after the nightly start time
+        // then it should appear on the dashboard results for the
+        // subsequent day.
+        $build_datetime = new \DateTime($date);
+        $build_start_timestamp = $build_datetime->getTimestamp();
+        $nightly_start_timestamp = strtotime($this->NightlyTime);
```

Comment:
@zackgalbreath, is this also in the local CDash server timezone and not UTC?

Reply:
`NightlyTime` is set by the project admin when they create or edit their project. It can have an explicit time zone, eg `01:00:00 UTC`.

If a time zone isn't set here then it will be interpreted relative to the time zone set in `config.local.php` (UTC by default).